src/styles/tokens.css

```
@@ -0,0 +1,54 @@
+        --line: rgba(20, 18, 12, 0.1);
+        --line-2: rgba(20, 18, 12, 0.06);
+        --accent: #0f6b66; /* teal */
+        --accent-ink: #ffffff;
+        --shadow:
+            0 1px 0 rgba(20, 18, 12, 0.04),
+            0 8px 24px -16px rgba(20, 18, 12, 0.18);
+        --radius: 0.875rem;
+        --radius-sm: 0.5rem;
+
+        --font-sans: "Geist Variable", ui-sans-serif, system-ui, sans-serif;
+        --font-mono: "Geist Mono Variable", ui-monospace, "JetBrains Mono", Menlo, monospace;
```

Stylelint (17.11.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)